_schemaV2.graphql

```diff
@@ -1,5 +1,3 @@
-directive @principalField on FIELD
```

Comment:
cc @mzikherman does this look right?

Reply:
nope! A [recent upgrade](https://github.com/artsy/metaphysics/pull/1933) to `graphql-tools` started including these. Is it possible this was run not on latest `master` or without a installing node modules?